ConnectorArchicad/ConnectorArchicad/Communication/HttpCommandExecutor.cs

```diff
@@ -42,10 +42,17 @@ private static TResponse DeserializeResponse<TResponse>(string obj)
         )
       )
       {
+        bool log = false;
+
         AddOnCommandRequest<TParameters> request = new AddOnCommandRequest<TParameters>(commandName, parameters);
 
         string requestMsg = SerializeRequest(request);
-        Console.WriteLine(requestMsg);
+
+        if (log)
+        {
+          Console.WriteLine(requestMsg);
```

Comment:
Did we really need to keep this log message as something you can turn on and off?